pyproject.toml

```diff
@@ -13,11 +13,11 @@ requests = "^2.31.0"
 llama-cpp-python = "^0.2.56"
 tiktoken = "^0.6.0"
 drain3 = "^0.9.11"
+progress = "*"
```

Comment:
Version should be pinned to a sensible default.

Reply:
This is a sensible default.
Poetry suggested I use' 1.6`, But I disagree with that. It works with 1.5, too (and any lower version), And it will likely continue to work with 2.0, too. IMO, there is no need to restrict it; it would just add an additional chore in the future.